Tests/scripts/update_conf_json.py

```diff
@@ -0,0 +1,112 @@
+import os
```

Comment:
add the following at the start:
```
#!/usr/bin/env python3
```
and make the file executable. That way it is possible to call the file directly.